### ISSUE TITLE
[GUI] Trailing [/B] tag when an addon emboldens a label as well as a skin.

### ISSUE THREAD

**Issue by Hitcher** [xbmc/xbmc#25263]
### Describe the bug
Here is a clear and concise description of what the problem is:

When an addon provides a bold label to a skin that then uses `[B]...[/B]` tags to embolden its label the output ends up with a single closing bold tag - `[/B]`

## Expected Behavior
Here is a clear and concise description of what was expected to happen:

Any extra tags introduced from a combination of bold addon label + bold skin label should be ignored and not displayed.

## Actual Behavior

Extra tag is displayed.

### To Reproduce

1. Edit lines 47 & 75 of View_55_WideList.xml in Estuary from `<label>$VAR[ListLabelVar]</label>` to `<label>[B]$VAR[ListLabelVar][/B]</label>`
2. Install iplayer www addon
3. Run iplayer www addon and open 'Highlights'
4. See the tag `[/B]` after bold labels from the addon

### Debuglog

The debuglog can be found here:

### Screenshots 
Here are some links or screenshots to help explain the problem:

![screenshot00003](https://github.com/xbmc/xbmc/assets/133808/6954e55b-acbe-40f4-a6c5-9873038766ae)

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

### Your Environment
Used Operating system:

 - [ ] Android
 - [ ] iOS
 - [ ] tvOS
 - [ ] Linux
 - [ ] macOS
 - [ ] Windows
 - [ ] Windows UWP

 - Operating system version/name:
 - Kodi version:

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by Fuchs246** [comment]
This seems to be a very old problem.

look here:
https://forum.kodi.tv/showthread.php?tid=333859&pid=2755961#pid2755961

I think the skin should always take precedence over the addon when it comes to text design.
It is time to finally correct this mistake.

**Comment by Hitcher** [comment]
> I think the skin should always take precedence over the addon when it comes to text design.

I agree but addon developers also like to control how things are displayed so a solution in core to ignore double tags would be the best option to keep both sides happy.

**Comment by Fuchs246** [comment]
> I agree but addon developers also like to control

Unfortunately, I can't completely agree with that. The addon developer should only provide the info labels. Only the skin developer should decide on the appearance.

**Comment by Hitcher** [comment]
Still valid.

**Comment by Hitcher** [comment]
Fixed in https://github.com/xbmc/xbmc/pull/28280
